# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../arcload/debug.patch
# Copyright (C) 2020 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- arcload-0.5/arcgrub/arcfs.c.vanilla	2020-07-05 14:09:13.737424442 +0200
+++ arcload-0.5/arcgrub/arcfs.c	2020-07-05 14:58:08.849425865 +0200
@@ -30,7 +30,7 @@
 	FILEINFORMATION info;
 
 	if(ArcOpen((char *)name, OpenReadOnly, &(data->id)) != ESUCCESS) {
-		grub_error(GRUB_ERR_FILE_NOT_FOUND, "ArcOpen failed");
+		grub_error(GRUB_ERR_FILE_NOT_FOUND, "ArcOpen failed: %s", name);
 		return grub_errno;
 	}
 	data->pos = 0;
